core/src/main/java/nu/yona/server/analysis/entities/GoalConflictMessage.java

```diff
@@ -125,22 +119,17 @@ public static GoalConflictMessage createInstanceFromBuddy(UUID relatedUserAnonym
 	{
 		if (origin == null)
 		{
-			throw GoalServiceException.originCannotBeNull();
+			throw new IllegalArgumentException("origin cannot be null");
```

Comment:
I see. I agree to not growing the change set any further. Logged [YD-151](http://jira.yona.nu/browse/YD-151).